### PR TITLE
Landing page new version - 1

### PR DESCRIPTION
**Pull request by sanjayk0508**
### Landing page preview

https://github.com/notaryproject/notaryproject.dev/assets/102804548/def070c0-2604-49cd-b73e-0e0bb899a7de

### This PR contains
- **Redesign of Landing page**: The landing page has been redesigned with consistent colors and a new user interface.
- **Refactor**:  Thoroughly restructured each section of the home page to enhance the user experience.
- **Mobile responsive**: The interface is now fully responsive, ensuring a seamless experience for users accessing the application on small devices.

### CODE REVIEW

**Review by zr-msft — APPROVED** [review]
LGTM

File: content/en/_index.html [hunk 1]
```diff
@@ -13,247 +13,1763 @@
...
-    <p class="width-60 is-centered font-size-md">
-      Signing and validating software artifacts, ensure they have not been tampered with and provide security policies
-      to determine which validated artifacts are allowed to be used in your systems
+
+<!-- {{< /blocks/cover >}} -->
+<div class="wf-block">
+  <div class="sn-cont">
+    <h1 class="sn-hd">Scenarios we fit and problems we solve for</h1>
+    <p class="sn-desc">
+      Sign and validate software artifacts to ensure integrity, prevent
```
Comment (FeynmanZhou) [inline content/en/_index.html:1281]: Sign and validate software artifacts, and ensure they have not been tampered with. Provide security policies to determine which validated artifacts are allowed to be used in your systems

File: content/en/_index.html [hunk 2]
```diff
@@ -13,247 +13,1763 @@
...
-<section class="feature-container">
-  <div class="feature-text-container text-center">
-    <h1 class="title">Why the Notary Project is unique</h1>
-    <p class="width-60 is-centered font-size-md">
-      The Notary Project is aiming to provide enterprise-grade solutions and cross-industry standards for securing
-      software supply chain
+
+<div class="un-n">
+  <div class="u-cont">
+    <h4 class="u-cont-h">Why Notary is unique?</h4>
```
Comment (FeynmanZhou) [inline content/en/_index.html:1360]: Why the Notary Project is unique?

File: content/en/_index.html [hunk 3]
```diff
@@ -13,247 +13,1763 @@
...
+    <p class="u-cont-d">
+      Notary is aiming to provide enterprise-grade solutions and cross-industry
```
Comment (FeynmanZhou) [inline content/en/_index.html:1362]: The Notary Project is aiming to provide enterprise-grade solutions and cross-industry standards for securing software supply chain

File: content/en/_index.html [hunk 4]
```diff
@@ -13,247 +13,1763 @@
...
+        <p class="desc-at-block">
+          AWS team is using and contributing to Notation, building the
+          cryptographic signing services for customers
+        </p>
+      </div>
     </div>
-    <div class="text-container text-left">
-      <p>Zot registry supports store Notation signature as OCI artifacts</p>
+    <div
+      class="horizontalCard flex gap-md space-between is-align-items-center"
```
Comment (FeynmanZhou) [inline content/en/_index.html:1619]: The layout effect looks not correct on my 14' inch screen. 
image

File: layouts/partials/footer.html [hunk 5]
```diff
@@ -1,22 +1,395 @@
...
+      <li>
+        <a href="https://app.slack.com/client/T08PSQ7BQ/CQUH8U287/">Slack</a>
+      </li>
+      <li>
+        <a href="https://hackmd.io/_vrqBGAOSUC_VWvFzWruZw"
+          >Notary Project Meeting doc</a
+        >
+      </li>
+      <li>
+        <a href="https://hackmd.io/wii3-L8ZQZ-U3ET0XNY8Gg">TUF Meeting doc</a>
```
Comment (FeynmanZhou) [inline layouts/partials/footer.html:297]: Please remove `TUF Meeting doc`. We no longer maintain TUF in the Notary Project community.

File: layouts/partials/footer.html [hunk 6]
```diff
@@ -1,22 +1,395 @@
...
+  </div>
+
+  <div class="footer-list-com">
+    <h4>Communities</h4>
+    <ul>
+      <li>
+        <a href="https://app.slack.com/client/T08PSQ7BQ/CQUH8U287/">Slack</a>
+      </li>
+      <li>
+        <a href="https://hackmd.io/_vrqBGAOSUC_VWvFzWruZw"
```
Comment (FeynmanZhou) [inline layouts/partials/footer.html:292]: Please make sure all external links will be opened and directed to a separate window.

File: layouts/partials/footer.html [hunk 7]
```diff
@@ -1,22 +1,395 @@
...
+        >
+        <a
+          href="https://www.linuxfoundation.org/privacy"
+          target="_blank"
+          rel="noopener"
+          class="f-text"
+          >Privacy</a
+        >
+        <a
+          href="https://www.linuxfoundation.org/trademark-usage"
```
Comment (FeynmanZhou) [inline layouts/partials/footer.html:343]: These four links should be placed in one line. This is the preview on my 14' inch screen. Please also add a YouTube icon.
image

File: layouts/shortcodes/home/cncf.html [hunk 8]
```diff
@@ -1,5 +1,13 @@
-{{ $src := "https://raw.githubusercontent.com/cncf/artwork/master/other/cncf/horizontal/color/cncf-color.png" -}}
+{{ $src :=
+"https://raw.githubusercontent.com/cncf/artwork/master/other/cncf/horizontal/color/cncf-color.png"
+-}}
 <section class="home--top-section">
-  <h4>Notary project is a <a href="https://cncf.io">CNCF</a> incubating project</h4>
-  <img class="cncf-logo" src="{{ $src }}" alt="Cloud Native Computing Foundation logo">
+  <h4 class="in-project">
+    Notary project is a <a href="https://cncf.io">CNCF</a> incubating project
```
Comment (FeynmanZhou) [inline layouts/shortcodes/home/cncf.html:6]:
```suggestion
    Notary Project is a <a href="https://cncf.io">CNCF</a> incubating project
```

Comment (FeynmanZhou) [reply layouts/shortcodes/home/cncf.html:6]: @sanjayk0508  Please note the project brand and name change in https://github.com/notaryproject/.github/issues/35

**Review by FeynmanZhou — CHANGES_REQUESTED** [review]
PTAL

**Comment by sanjayk0508** [comment]
Thanks, @FeynmanZhou. I have updated the interface to be consistent with the requirements, ensuring it is compatible with smaller screens and that links open in new tabs. Additionally, I revised the content in the `feature` section by avoiding writing it in the first person. Please review it now.

**Comment by yizha1** [comment]
Thanks @sanjayk0508 . I am not the UI expert, so my comment is mainly from a user point of view:
- The new page is too long, I have to scroll down the page several times to see all the content. 
- The background blue seems too bright for me.
- Suggest removing the roadmap page, since the `roadmap` repo is not actively maintained now. We can add the page back once we know how the roadmap is maintained.

**Comment by sanjayk0508** [comment]
> Thanks @sanjayk0508 . I am not the UI expert, so my comment is mainly from a user point of view:
> 
> * The new page is too long, I have to scroll down the page several times to see all the content.
> * The background blue seems too bright for me.
> * Suggest removing the roadmap page, since the `roadmap` repo is not actively maintained now. We can add the page back once we know how the roadmap is maintained.

Thanks, @yizha1, for your review. I will certainly take it into consideration. If other maintainers share a similar viewpoint, we will update the UI accordingly. I will try to gather feedback in the upcoming meeting.

**Comment by sanjayk0508** [comment]
@FeynmanZhou @zr-msft PTAL now.

**Comment by sanjayk0508** [comment]
@FeynmanZhou @zr-msft PTAL now.

**Review by zr-msft — APPROVED** [review]

**Review by zr-msft — CHANGES_REQUESTED** [review]
Section at the top now cuts off text/ui in the docs section. See image:

Screenshot 2023-08-02 at 3 35 45 PM

**Comment by sanjayk0508** [comment]
> Section at the top now cuts off text/ui in the docs section. See image:
> 
> Screenshot 2023-08-02 at 3 35 45 PM

Fixed the issue. The entire `Docs` structure comes under the `theme` folder which is unedited, so I have made adjustments in the navbar and banner. PTAL

**Review by zr-msft — APPROVED** [review]

**Pull request closed without merging**